src/ripplegw/waveforms/IMRPhenomXAS.py

```
@@ -14,7 +14,11 @@
 amp_uneqspin_indx = 36
 
 
-def get_inspiral_phase(fM_s: Array, theta: Array, phase_coeffs: Array) -> Array:
+def get_inspiral_phase(
+    fM_s: Float[Array, " n_freq"],
```

Ruff (0.15.18)

[error] Syntax error in forward annotation: Expected an expression

(F722)